src/style/global.scss

```diff
@@ -10,7 +10,7 @@
 
   font-size: $iui-font-size;
   line-height: $iui-line-height;
-
+  scroll-behavior: smooth;
```

Comment:
If this applies to all components, I think this could lead to unintended behavior in some places.

For example, in Select, we automatically scroll to the selected option. Setting `smooth` would cause an unnecessary "delay" while the animation is happening.

Reply:
Oh and it would also affect user apps. So this is definitely not the right place.

Reply:
I've removed this for now, will rely on the individual app teams to implement this on their own as needed.